test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -0,0 +1,340 @@
+require 'test_helper'
+
+class RemoteStripeIntentsTest < Test::Unit::TestCase
```

Comment:
it's the same as in the other PR, right?

Reply:
@Azdaroth yeah, that's why I linked the PR. Here I changed only Ogone and it's specs.